dashboard/app/models/script_level.rb

```diff
@@ -725,6 +726,43 @@ def add_variant(new_level)
     end
   end
 
+  def get_example_solutions(current_user, section=nil)
+    level_example_links = []
+
+    level = levels.first
+
+    if level.try(:examples).present? && (current_user.authorized_teacher? || script.csf?) # 'solutions' for applab-type levels
```

Comment:
Could we add some tests for this method, particularly the logic around the different cases here?

Reply:
Added!